### ISSUE TITLE
Potential issue with coordinates?

### ISSUE THREAD

**Issue by AlucardTheRed9** [goatfungus/NMSSaveEditor#820]
Im not sure if that is an issue or not, but for some reason the first glyph does not change when changing coordinates in the editor. I have not tried to teleport and see if it still works as intended, and its just a UI bug, or if it really does not change to the correct address. All the other glyphs (and portal addr numbers) change normally, EXCEPT the first one. That one stays the same number no matter what, which is 0. Really odd.

![Screenshot (154)](https://github.com/goatfungus/NMSSaveEditor/assets/96874517/21c9b6bd-ecb2-496a-a44e-e1f49e1ec1fb)

**Comment by goatfungus** [comment]
This is a common complaint.

The coordinate viewers main function is to warp you to a different system, and it can ONLY do that if the first digit is ZERO. This is because 0 represents "in space", where each other number means "on planet surface". I would have loved to be able to make it warp to a planet surface as well, but since there is no way to determine where the surface of the planet is, you would just end up either floating in mid-air or inside the planet itself.

In conclusion, this is not an issue with the editor, it's a valid limitation.

**Comment by AlucardTheRed9** [comment]
Oh, no, thats not a concern to me. Im fine with warping to space. Its
literally just a matter of flying to a planet you want from there (which,
if someone cant do, theyre a lazy cunt). I simply did not understand why
that odd behaviour in the editor was happening, and whether it had a
function or not. I later found out that the first digit/symbol somehow
represents the teleporter (or is supposed to), corresponding to the planet
that it is on, so if 0 is space, then I guess it makes sense for universal
warping...? I also later tried to teleport with it, and it seemed to work
fine, dispelling my doubts. Only once did it warp me to a wrong system
somehow, requiring to enter the coordinates again, but that may have been
my error again.
Anyway, I guess its solved now. Thanks. Sorry fo the waste of time. 🙂
